src/cli.zig

```diff
@@ -323,6 +362,18 @@ test "parse_size" {
     try expectEqual(@as(u64, 1000 * kib), parse_size("  1000  kb "));
 }
 
+/// Given a limit ilke `10GiB`, return the maximum power-of-two count of `T`s
+/// that can fit in the limit.
+fn parse_size_to_count(comptime T: type, string: []const u8) u32 {
+    const byte_size = parse_size(string);
+    const count = math.floorPowerOfTwo(u64, byte_size / @sizeOf(T));
```

Comment:
Instead of `/`, use `@divFloor()` — we try to use `@divFloor()` and `@divExact` in place of `/` to be explicit. (`@divExact()` will panic if the division has a remainder).

Reply:
Do we want to collect bits of advice like this (also, stuff like "blank like after defre") into some document alongside `TIGER_STYLE.md`?

Reply:
Sure.

Reply:
Ah, actually this on is already in TIGER_STYLE.md, let me then start with re-reading that!